.gitignore

```diff
@@ -0,0 +1,2 @@
+
+.DS_Store
```

Comment:
I guess I didn't have a `.gitignore` yet, because I have some things like this in my global .gitignore. I will accept this and merge with this file, but often if you are doing a pull request on an open-source project, you would want to only commit what is relevant to the change you are making and not extra things that the owner of the project may not want.

Let me know if you need any clarification :)

Thanks!